spec/unit/puppet_x/puppetlabs/scheduled_task/trigger_spec.rb

```diff
@@ -924,6 +940,18 @@
       end
     end
 
+    [
+      { :ole_type => 'IBootTrigger', :Type => V2::Type::TASK_TRIGGER_BOOT, },
+    ].each do |trigger_details|
+      it "should convert an #{trigger_details[:ole_type]} instance" do
+        # stub is not usable outside of specs (like in DEFAULT_V2_ITRIGGER_PROPERTIES)
+        iTrigger = stub(DEFAULT_V2_ITRIGGER_PROPERTIES.merge(trigger_details))
+        iTrigger.Repetition.stubs(:Interval).returns('')
```

Comment:
I don't think the stubs here are needed, are they?

Reply:
Given the prior changes to the way Duration was getting checked these stubs were required. The current mock returns Interval and Duration as hash keys and the way I had changed the code required a mock of a method or the tests broke. Since I have reverted those changes based on comments above these stubs are indeed no longer needed and I am removing them.